### PR TITLE
Refactor case court report specs

### PR DESCRIPTION
**Pull request by napster235**
### What github issue is this PR for, if any?
There is no open issue for this PR

### What changed, and why?
Refactored case court report specs

### How will this affect user permissions?
- Volunteer permissions: Doesn't affect
- Supervisor permissions: Doesn't affect
- Admin permissions: Doesn't affect

### How is this tested? (please write tests!) 💖💪
No need for extra specs

### Feelings gif (optional)
What gif best describes your feeling working on this issue? https://giphy.com/
How to embed:
`alt text`

### Feedback please? (optional)
We are very interested in your feedback! Please give us some :) https://forms.gle/1D5ACNgTs2u9gSdh9

### CODE REVIEW

**Comment by napster235** [comment]
@FireLemons @compwron I opened this PR with similar changes to #4473 because I noticed the same patter here aswell.

**Review by compwron — APPROVED** [review]
looks good :) let's merge when CI is green

**Comment by napster235** [comment]
@compwron I think we can merge this one since the CI build has been fixed

**Comment by napster235** [comment]
@compwron any idea why the linter fails? When running `bundle exec standardrb --fix ` nothing it getting fixed. Any idea how to fix? Thanks

File: spec/models/case_court_report_spec.rb [hunk 1]
```diff
@@ -225,29 +284,39 @@
...
 
+      it "contains the court order text" do
         expect(document_inspector.word_list_document_contains?(court_order_unimplemented.text)).to eq(true)
+      end
+
+      it "contains the exact value of 'Unimplemented'" do
         expect(document_inspector.word_list_document_contains?("Unimplemented")).to eq(true)
+      end
 
+      it "contains the court orde text" do
```
Comment (FireLemons) [inline spec/models/case_court_report_spec.rb:307]: order

**Pull request merged**